models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql

```diff
@@ -0,0 +1,14 @@
+{{ config(alias='order_rewards',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "cow_protocol",
+                                    \'["bh2smith"]\') }}'
+)}}
+
+-- PoC Query here - https://dune.com/queries/1752782
+select
+    tx_hash,
+    data.amount as cow_reward,
+    cast(data.surplus_fee as double) as surplus_fee,
+    order_uid
+from cowswap.raw_order_rewards
```

Comment:
This is not using a ref call but CI does not complain... Not sure yet how to reference community sources...

Reply:
According to this:

https://github.com/duneanalytics/spellbook/blob/ad2b91fef757f12a885fd2b71df293319e047f68/models/labels/flashbots/labels_flashbots_ethereum.sql#L15-L15

I should use source(`cowswap`, 'raw_order_rewards')

Reply:
Fixed